### PR TITLE
Complex nerd pull request

### PR DESCRIPTION
**Pull request by Austing767**
Makes a circle that represents a face
then adds glasses
and makes a reference to a meme about nerds with the comment function.

### CODE REVIEW

File: hw03_gatewooda.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    happy.penup()
+    happy.goto(20, -radius)
+    happy.pendown()
+
+    # Draw a circle
+    for _ in range(360):
+        happy.forward(radius * 2 * math.pi / 360)     # this might be why
+        happy.left(1)
+
+
```
Comment (sheggen) [inline hw03_gatewooda.py:36]: Only 2 blank lines.

File: hw03_gatewooda.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        found.right(90)
+
+    found.forward(50)
+
+    for i in range(2):
+        found.right(225)
+        found.forward(50)
+
+
+def comment():
```
Comment (sheggen) [inline hw03_gatewooda.py:75]: Docstrings for all functions.

File: hw03_gatewooda.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    text = turtle.Turtle()
+    text.penup()
+    text.goto(-60,0)
+    text.write("erm actually", align="right", font=("calibri", 24, "normal"))
+    text.hideturtle() # id like to add more details like a mouth to add to the complex requirement
+
+def main():
+    """
+    Docstring for main. Should describe the main functionality of this file.
```
Comment (sheggen) [inline hw03_gatewooda.py:84]: Replace my help text with an actual docstring. What's this program's purpose?

File: hw03_gatewooda.py (hunk 1 above)
Comment (Austing767) [reply hw03_gatewooda.py:36]: I have changed where every function definition has two blank spaces between the previous function.

File: hw03_gatewooda.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    happy.speed(0)
+    happy.color("#087209")
+
+    # Move forward to the position where the face should start
+    happy.penup()
+    happy.goto(20, -radius)
+    happy.pendown()
+
+    # Draw a circle
+    for _ in range(360):  # In the range of 360 degrees radius is doubled than multiplied by pi
```
Comment (sheggen) [inline hw03_gatewooda.py:34]: This is unusual. Seems like you might have pulled the idea off the web. Don't forget to cite sources.

Comment (Austing767) [reply hw03_gatewooda.py:34]: I had used a similar design for Hw02 and expanded on it however I did use this website provided down below, for inspiration. 
Screenshot (1)
https://www.geeksforgeeks.org/draw-circle-in-python-using-turtle/

I've added both to the acknowledgements :)

**Pull request merged**